default/src/components/signer.js

```diff
@@ -179,9 +178,16 @@ const SendASA = () => {
         ledger: CHAIN_NAME,
       })) ?? [];
 
-    setAccounts(algoSignerAccts.map(acc => acc.address));
     if (algoSignerAccts.length) {
+      // TODO: update fromAddress to current active account in https://www.pivotaltracker.com/story/show/178760753
```

Comment:
can we not do this in current PR ? probably we can get account using a function in algosigner or by reading state

Reply:
got it. https://github.com/PureStake/algosigner/issues/252